cell_classification/segmentation_data_prep.py

```diff
@@ -167,12 +174,11 @@ def prepare_example(self, data_folder, marker):
         mplex_img = self.get_image(data_folder, marker)
         mplex_img /= self.normalization_dict[marker]
```

Comment:
I think eventually it will make more sense to have this normalization happen in a different function. Probably we'll want to avoid re-normalizing the data every time we load it in, and instead normalizing once and then loading the normalized data each time would make more sense. Not necessary for the first pass though, so if you want to just make an issue about it and we can fix it later once it becomes a bottleneck that's totally fine

Reply:
Normalizing is really cheap once the `normalization_dict` is calculated and we don't need to have a copy of the dataset stored on disc. I put it into the backlog #6.